packages/engine/tests/units/state/edit/dot/bool_array_fixed/src/behaviors/test.js.json

```diff
@@ -0,0 +1,37 @@
+{
+  "keys": {
+    "b1": {
+      "type": "list",
+      "nullable": false,
```

Comment:
What was the reasoning on setting a lot of these (as in across the suite, not just this case) as not nullable?

Reply:
Hm. I think in the future we should revisit how we test nullable fields in general. But for now it's better there's a mix, even if a little bit of a random one

Reply:
I tried to reasonably apply `nullable` everywhere, but I'm pretty sure, there are unneeded `"nullable": true`s. Generally, I agree, but not a suitable task for the near future. We should think of a more general testing strategy.